LanguageFeatures/Patterns/variable_declaration_A01_t01.dart

```diff
@@ -54,7 +54,7 @@ main() {
 // [analyzer] unspecified
 // [cfe] unspecified
 
-  Map<String, num> {"pi": a4, ...} = {"pi": 3.14, "e": 2.71};
+  Map<String, num> {"pi": a4} = {"pi": 3.14, "e": 2.71};
 //                 ^^^^^^^^^^^^^^^
```

Comment:
Presumably the placement of '^^^^' is ignored for 'unspecified' errors, but we might want to delete those extra ^'s because that's the style otherwise.

Reply:
Thank you!

Reply:
Thanks! ;-)